tests/hazmat/primitives/test_ec.py

```diff
@@ -45,17 +46,18 @@ def _skip_ecdsa_vector(backend, curve_type, hash_type):
         curve_type()
     ):
         pytest.skip(
-            "ECDSA not supported with this hash {} and curve {}".format(
-                hash_type().name, curve_type().name
+            "ECDSA not supported with this hash {} and curve {}. "
+            "FIPS mode: {}".format(
```

Comment:
It's extremely annoying to need to scatter this everywhere. Can we add it to the backend repr and just include the backend in the skip or something?

Reply:
Our backends don't have a repr! (This is an issue I noticed earlier today, lol)